JSDemos/Demos/Toolbar/Multiline/React/styles.css

```diff
@@ -0,0 +1,54 @@
+.dx-resizable-handle::after {
+  content: "";
+  background-color: #337ab7;
+  position: absolute;
+  width: 12px;
+  height: 12px;
+  border: 1px solid #337ab7;
+  top: calc(50% - 7px);
+}
+
+.dx-resizable-handle-right::after {
+  right: -7px;
+}
+
+.dx-toolbar.dx-toolbar-multiline .dx-toolbar-item {
+  margin-bottom: 5px;
+}
+
+.dx-toolbar-menu-section .menu-separator-container {
```

Comment:
Seems, that .menu-separator-container class is redundant. I can't reproduce the scenario where it needs. Do you have a sample with configuration?

Reply:
We have 5px height container for Separator div without this rule. This issue is reproduced in React, Vue and Angular.

Reply:
It issue reproduced in react, angular, vue only. We added additional container with separator class.